src/less/icon/foreground/ds6/foreground.less

```diff
@@ -81,8 +81,8 @@ svg {
     }
 
     &.icon--info,
-    &.icon--information-filled {
-        .foreground-icon-image-mixin('information-filled');
+    &.icon--information-clear {
+        .foreground-icon-image-mixin('information-clear');
```

Comment:
Why are these flipping? Wouldn't they stay as the filled versions if that was the intent before?

Reply:
The original should still be there.  Its just adding the new one.  He added "filled" before.  Now he's adding "clear".   But the original should still exist.

Reply:
This is correct. 
Before these changes:
`icon-info` mapped to the clear icon. 
`icon-information` mapped to the colored icon

Now `icon-info` maps to the `icon-information-clear`. 
It was a mistake before and this is the remedy.

Reply:
I was confused by it for a bit too.  (And still could be wrong, obviously)

Reply:
`icon-info` mapping to the clear icon, and `icon-information` mapping to the coloured icon was a bad idea, in hindsight (I do believe there was some kind of reason for it though). Now that we have a major version we can name these more explicitly to avoid confusion.

Reply:
That's what this pr is for. We are just keeping the short names so they are backwards compatible. 
But we can remove the shorthand names if they cause confusion and  have no use

Reply:
Major versions should definitely be used for fixing things that caused confusion! Just make sure to call it out in the release notes.